.github/workflows/create-production-build.yml

```diff
@@ -0,0 +1,59 @@
+name: Build on Release Branch Merged into Deploy
+
+on:
+  push:
+    branches:
+      - deploy
+
+jobs:
+  build:
```

Comment:
consider doing a build if instead of the check merge step...
```
jobs: 
 build:
   if: contains(github.event.head_commit.message, 'Merge branch \'release/v') && contains(github.event.head_commit.message, 'into deploy')
  runs-on: ubuntu-latest
```
   and leave this whole part out:
   `- name: Check if merge commit from release branch`
    Not sure if that if condition is exactly right...

Reply:
im going to take out this file for now, and just do the one flow, so we can test that the other one is working